client/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "client",
+  "name": "@developer-dao-job-board/client",
```

Comment:
imo since these packages are `private` and won't be published (at least for the time being), it'd be easier to just leave them as `client` and `database`. `yarn workspace @developer-dao-job-board/client` vs `yarn workspace client` 😁

Reply:
Hahaha right! 😁 well I just thought it was cool XD But let's go with the shortest name :D